rust/azure_iot_operations_services/src/azure_device_registry.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Types for Azure Device Registry operations.
+// use crate::azure_device_registry::device_name_gen::adr_base_service::client as adr_base_service_gen;
```

Comment:
nit - sometimes the doc generator gets weird without an extra line
```suggestion

// use crate::azure_device_registry::device_name_gen::adr_base_service::client as adr_base_service_gen;
```